mindsdb/integrations/handlers/github_handler/github_handler.py

```diff
@@ -108,3 +110,31 @@ def native_query(self, query: str) -> StatusResponse:
         """
         ast = parse_sql(query, dialect="mindsdb")
         return self.query(ast)
+
+
+connection_args = OrderedDict(
+    repository={
+        "type": ARG_TYPE.STR,
+        "description": " GitHub repository name.",
+        "required": True,
+        "label": "Repository",
+    },
+    api_key={
+        "type": ARG_TYPE.PWD,
+        "description": "Optional GitHub API key to use for authentication.",
+        "required": False,
+        "label": "Api key",
+    },
+    github_url={
+        "type": ARG_TYPE.STR,
+        "description": "Optional GitHub URL to connect to a GitHub Enterprise instance.",
+        "required": False,
+        "label": "Github url",
+    },
+)
+
+connection_args_example = OrderedDict(
+    repository="mindsdb/mindsdb", 
+    api_key="ghp_z91InCQZWZAMlddOzFCX7xHJrf9Fai35HT7", 
```

Comment:
Is this randomly generated key or an actual value?

Reply:
@ZoranPandovski yes, is a random code.